code/modules/antagonists/grinch/abilities/murder.dm

```diff
@@ -1,6 +1,71 @@
+/datum/action/bar/private/icon/grinchFlatline
+	duration = 120
```

Comment:
whats 120

Reply:
no idea

Reply:
Then why put it there?

Reply:
affects duration and i dont know what its measured in but its a good length

Reply:
you need to notate it with a time define then
https://hackmd.io/@goonstation/docs/%2F%40goonstation%2Fcode#Time-Defines